extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -147,13 +147,20 @@ class AmpYoutube extends AMP.BaseElement {
     Services.videoManagerForDoc(this.element).register(this);
   }
 
+  getEmbedUrl() {
+    if (this.getCredentials_() === 'omit') {
+      return `https://www.youtube-nocookie.com/embed/${encodeURIComponent(this.videoid_ || '')}?enablejsapi=1`;
+    }
+    return `https://www.youtube.com/embed/${encodeURIComponent(this.videoid_ || '')}?enablejsapi=1`;
```

Comment:
please add a test to test-amp-youtube.js to ensure Url is right with credentials is set to omit.